README.md

```diff
@@ -4,5 +4,6 @@ To sign, open a pull request. Link your profile or email in the signature. Pleas
 
 Let's keep the tone firm, but professional.
 
+PC went far too far
```

Comment:
Please add your signature or, if you don't want to support this letter, refrain from changing README.md.

Reply:
Fixed